### PR TITLE
Improved French translation

### PR DESCRIPTION
**Pull request by ferrocarta**
Corrected "permalink" translation and added new translations

### CODE REVIEW

**Comment by DerDakon** [comment]
I can't comment on that, no idea about that language.

File: locales/fr_FR/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -466,7 +466,7 @@ msgid "Language"
 msgstr "Langue"
 
 msgid "Desktop version"
-msgstr ""
+msgstr "Version bureau"
```
Comment (xalopp) [inline locales/fr_FR/LC_MESSAGES/messages.po]: I think, that following translation would be nicer:
- `Désactiver la version mobile`
- `Site web`
- `Site pour PC`

Comment (ferrocarta) [reply locales/fr_FR/LC_MESSAGES/messages.po]: Yes these sound better, "Site pour PC" would be my preferred one.

Comment (mmoll) [reply locales/fr_FR/LC_MESSAGES/messages.po]: @ferrocarta so please add that change to your PR, preferably squashed into the existing commit.

**Comment by DerDakon** [comment]
Please squash the commits together.

File: locales/fr_FR/LC_MESSAGES/messages.po (hunk 1 above)
Comment (ferrocarta) [reply locales/fr_FR/LC_MESSAGES/messages.po]: Hopefully I did it right, this is my first attempt at using Github, please forgive me if I make mistakes.

**Comment by ferrocarta** [comment]
Sorry I don't know how to do that, I tried "git rebase -i HEAD~1" but have no idea what to do next (I am not familiar at all with git)

**Comment by mmoll** [comment]
Try `git rebase -i HEAD~2` and use `fix` instead of `pick` for the second commit and save the editor. Then use `git push -f`.

**Comment by ferrocarta** [comment]
Thanks, I believe the squash worked.

**Comment by DerDakon** [comment]
Seems like. You now have different committer and author information in that commit, no idea if that was intended.

Otherwise: for those people that understand French: please vote up/down for this so we can decide if this should be merged or not.

**Comment by ferrocarta** [comment]
It wasn't intended, perhaps because I used the online GitHub interface first, then the command line git to squash, but probably not so important.

**Comment by DerDakon** [comment]
git commit --amend --author="Your name <e@mail>"

**Comment by DerDakon** [comment]
Do you want to change the author? Then I could pick it.

**Comment by ferrocarta** [comment]
I changed it, it looks ok.

**Comment by DerDakon** [comment]
You probably need to push it ;) To check the full metadata locally: git show --format=fuller

**Comment by ferrocarta** [comment]
Thanks for your patience :) Maybe it is ok now?

**Comment by DerDakon** [comment]
Put it online.

**Comment by ferrocarta** [comment]
Thanks, I can already see the changes on the website. I'll limit my contribution to this for now because I don't have a sufficiently good understanding of the code, but it's a great site :)

**Pull request merged**